README.rst

```diff
@@ -7,7 +7,7 @@ Tribler
 
 |downloads_7_0| |downloads_7_1| |downloads_7_2| |downloads_7_3| |downloads_7_4|
 |downloads_7_5| |downloads_7_6| |downloads_7_7| |downloads_7_8| |downloads_7_9|
-|downloads_7_10| |downloads_7_11| |downloads_7_12| |downloads_7_13|
+|downloads_7_10| |downloads_7_11| |downloads_7_12| |downloads_7_13| |downloads_7_14|
```

Comment:
Tribler [doesn't have](https://github.com/Tribler/tribler/releases) a 7.14 release yet, and the latest release is [7.13.1](https://github.com/Tribler/tribler/releases/tag/v7.13.1). So, it should be `downloads_7_13_1` instead. This is just an internal name, so it does not affect the resulting page, but having the correct name can help avoid possible confusion when adding further releases. So, please accept the following suggestion:
```suggestion
|downloads_7_10| |downloads_7_11| |downloads_7_12| |downloads_7_13| |downloads_7_13_1|
```